yeonjae/baekjoon/스택/10773-제로.js

```diff
@@ -0,0 +1,32 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const T = Number(input[0]);
+  const stack = [];
+
+  for (let i = 1; i <= T; i += 1) {
+    const num = Number(input[i]);
+
+    if (num === 0) {
+      stack.pop();
+    } else {
+      stack.push(num);
+    }
+  }
+
+  const result = stack.reduce((acc, cur) => acc + Number(cur), 0);
+
+  console.log(result);
```

Comment:
```suggestion
const numList = input.slice(1).map(Number);
  const stack = [];

  numList.forEach((num) => {
    if (!num) stack.pop();
    else stack.push(num);
  });

  console.log(stack.reduce((acc, cur) => acc + cur, 0));
```
- 풀이 잘봤습니다! 🙇🏻‍♂️
- slice를 활용해 입력값을 가져오고 미리 형변환을 끝내면 추후에 형변환에 대하 고민이 없어지기 때문에 미리 입력값을 핸들링해서 활용하시는 것을 추천드립니다!
- 또한, forEach를 사용하시면 좀 더 깔끔한 코드가 가능합니다